alibi_detect/utils/saving.py

```diff
@@ -1120,144 +1103,159 @@ def load_tf_hl(filepath: str, model: tf.keras.Model, state_dict: dict) -> List[t
     -------
     List with loaded tf.keras models.
     """
-    model_dir = os.path.join(filepath, 'model')
+    model_dir = Path(filepath).joinpath('model')
     hidden_layer_kld = state_dict['hidden_layer_kld']
     if not hidden_layer_kld:
-        return None
+        return [None]
```

Comment:
Why the change to list return?

Reply:
The output type hint is `-> List[tf.keras.Model]` i.e. `load_tf_hl` returns a list of models. Returning a list of `[None]` gets rid of the type hint warning here. Perhaps `return []` would actually be more suitable here?

Reply:
I see, `[]` makes sense to me.